### PR TITLE
grammar change to install.html

### PR DESCRIPTION
**Pull request by jlafleur93**
Grammar seems to need a slight update, I created an issue here https://github.com/ebiten/ebiten.org/issues/70
the word succeeded in the sentence  seems out of place.

### CODE REVIEW

**Comment by hajimehoshi** [comment]
Thanks. As the README says, could you modify the file in content and run `go generate`?

File: docs/documents/install.html [hunk 1]
```diff
@@ -197,7 +197,7 @@ <h2>Ebiten</h2>
 <p lang="ja"><code>go get</code> は Ebiten のコードを取得します。 <code>go run</code> は Go ファイルをコンパイルし、即座に実行します。最初のコード取得とコンパイルは少々時間がかかるかもしれないため、しばらくお待ち下さい。必要ならば <code>-x</code> をつけることで、コンパイル処理中に何が行われているのかを見ることができます。</p>
 <p lang="en">Note that <code>-tags=example</code> is needed to execute the examples as the Ebiten example packages require the build tag.</p>
 <p lang="ja"><code>-tags=example</code> は Ebiten のサンプルを実行するために必要です。 Ebiten サンプルのパッケージがビルドタグを必要とするためです。</p>
-<p lang="en">If you see the window with a rotating Gophers picture, congratulations! You have succeeded to install Ebiten.</p>
+<p lang="en"> If you see this window with a rotating Gophers image, congratulations! <br> <br> You have sucessfully installed Ebiten! </p>
```
Comment (hajimehoshi) [inline docs/documents/install.html]: Create a new paragraph instead of using `<br>` here.

**Comment by jlafleur93** [comment]
> Thanks. As the README says, could you modify the file in content and run `go generate`?

Yes, I tested it on local, and it seemed to work fine. 
Also thanks for making this Hajime.

**Comment by hajimehoshi** [comment]
Yeah so the file in content should be in the PR.

Thank you for using Ebiten!

**Comment by hajimehoshi** [comment]
ping

**Comment by jlafleur93** [comment]
edit: nvm

should be good now

**Comment by hajimehoshi** [comment]
Again,

> Yeah so the file in content should be in the PR.

Thanks,

**Comment by hajimehoshi** [comment]
1. Modify `contents/documents/install.html`
2. Run `go generate`
3. Include both files in the commit

Thanks...

**Comment by jlafleur93** [comment]
Sorry, Hajime... 

Should be all good now?

**Comment by hajimehoshi** [comment]
> Include both files in the commit

Please...

**Comment by hajimehoshi** [comment]
If you are still not sure, is it ok that I will commit your change?

**Comment by jlafleur93** [comment]
> If you are still not sure, is it ok that I will commit your change?

Yeah its fine

**Pull request closed without merging**